### PR TITLE
Admin route

### PR DESCRIPTION
**Pull request by Gfmunoz**
Added admin-route in __init__.py and created an HTML template for the admin-route.

### CODE REVIEW

**Review by josueal1 — CHANGES_REQUESTED** [review]
@Gfmunoz Must add `@login_required` python decorator right under to protect from unlogged-in users

& update route to `/admin`

File: app/__init__.py [hunk 1]
```diff
@@ -48,6 +48,11 @@ def index():
     def home():
         return render_template('home.html')
 
+    #new admin route.
+    @app.route('/admin-route')
```
[the remark this answers is not in the document]
Comment (Gfmunoz) [reply app/__init__.py]: Done :)

File: app/templates/admin.html [hunk 2]
```diff
@@ -0,0 +1,9 @@
+<html lang="en"
+<head>
+    <title>Admin Route</title>
+</head>
+
+<body>
+    <p>Admin route</p>
+</body>
```
Comment (josueal1) [inline app/templates/admin.html:8]: @hlopezes14 Let's add chart onto this secure "admin.html" which will be served login secured route. Just `git fetch` and `git merge` latest master into your working branch

**Review by josueal1 — APPROVED** [review]
lgtm !

**Pull request merged**